sdk/core/System.Net.ClientModel/api/System.Net.ClientModel.net6.0.cs

```diff
@@ -0,0 +1,72 @@
+namespace System.Net.ClientModel
+{
+    public static partial class ModelReaderWriter
```

Comment:
Adding for tracking that we are required to review these APIs with the BCL team prior to GA'ing the package.